arc/conformance/plugin/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM k8s.gcr.io/build-image/debian-base:bullseye-v1.1.0
+ARG KUBE_VERSION=v1.21.2
+ARG TARGETARCH
+
+# install dependencies
+RUN apt-get update -y && apt-get upgrade -y
+RUN DEBIAN_FRONTEND=noninteractive TZ=America/Los_Angeles apt-get install python3-pip bash -y
+RUN python3 -m pip install junit_xml
+
+# install azcli
+RUN apt-get update && apt-get -y --allow-change-held-packages upgrade && \
```

Comment:
we don't need to run update again, and looks like we are installing python3-pip twice? can we combine these

Reply:
refactored